pkgs/by-name/ar/archivebox/package.nix

```diff
@@ -1,104 +1,185 @@
-    requests
-    w3lib
-    yt-dlp
+  dependencies =
+    with python.pkgs;
+    [
+      # Core Libraries
+      django
+      (django-ninja.overridePythonAttrs { doCheck = false; })
+      django-extensions
+      mypy-extensions
+      typing-extensions
+      channels
+      daphne
+      django-signal-webhooks
+      django-admin-data-views
+      django-object-actions
+      django-charid-field
+      django-pydantic-field
+      django-jsonform
+      django-stubs
+      django-huey
+      django-huey-monitor
+      # Helper Libraries
+      pluggy
+      requests
+      dateparser
+      tzdata
+      feedparser
+      w3lib
+      rich
+      rich-argparse
+      ulid-py
+      typeid-python
+      psutil
+      supervisor
+      python-crontab
+      croniter
+      ipython
+      py-machineid
+      python-benedict
+      pydantic-settings
+      atomicwrites
+      django-taggit
+      base32-crockford
+      platformdirs
+      pydantic-pkgr
+      pocket
+      sonic-client
+      yt-dlp
+
+    ]
+    ++ python.pkgs.python-benedict.optional-dependencies.io
+    ++ python.pkgs.python-benedict.optional-dependencies.parse;
+
+  optional-dependencies = {
+    ldap = with python.pkgs; [
+      python-ldap
+      django-auth-ldap
+    ];
+    debug = with python.pkgs; [
+      django-debug-toolbar
+      djdt-flamegraph
+      ipdb
+      requests-tracker
+      django-autotyping
+    ];
+  };
+
+  nativeInstallCheckInputs = [
+    versionCheckHook
   ];
 
-  makeWrapperArgs = [
-    "--set USE_NODE True" # used through dependencies, not needed explicitly
-    "--set READABILITY_BINARY ${lib.meta.getExe readability-extractor}"
-    "--set MERCURY_BINARY ${lib.meta.getExe postlight-parser}"
-    "--set CURL_BINARY ${lib.meta.getExe curl}"
-    "--set RIPGREP_BINARY ${lib.meta.getExe ripgrep}"
-    "--set WGET_BINARY ${lib.meta.getExe wget}"
-    "--set GIT_BINARY ${lib.meta.getExe git}"
-    "--set YOUTUBEDL_BINARY ${lib.meta.getExe yt-dlp}"
-  ] ++ (if (lib.meta.availableOn stdenv.hostPlatform chromium) then [
-    "--set CHROME_BINARY ${chromium}/bin/chromium-browser"
-  ] else [
-    "--set-default USE_CHROME False"
-  ]);
+  pythonImportsCheck = [ "archivebox" ];
+
+  makeWrapperArgs =
+    [
+      "--set USE_NODE True" # used through dependencies, not needed explicitly
+      "--set READABILITY_BINARY ${lib.meta.getExe readability-extractor}"
+      "--set MERCURY_BINARY ${lib.meta.getExe postlight-parser}"
+      "--set CURL_BINARY ${lib.meta.getExe curl}"
+      "--set RIPGREP_BINARY ${lib.meta.getExe ripgrep}"
+      "--set WGET_BINARY ${lib.meta.getExe wget}"
+      "--set GIT_BINARY ${lib.meta.getExe git}"
+      "--set YOUTUBEDL_BINARY ${lib.meta.getExe yt-dlp}"
+    ]
+    ++ (
+      if (lib.meta.availableOn stdenv.hostPlatform chromium) then
+        [
+          "--set CHROME_BINARY ${chromium}/bin/chromium-browser"
+        ]
+      else
+        [
+          "--set-default USE_CHROME False"
+        ]
+    );
```

Comment:
```suggestion
      "--set YOUTUBEDL_BINARY ${lib.meta.getExe yt-dlp}"
      (
        if (lib.meta.availableOn stdenv.hostPlatform chromium)
        then "--set CHROME_BINARY ${lib.getExe' chromium "chromium-browser"}"
        else "--set-default USE_CHROME False"
      )
    ];
```

Reply:
There's no need to use `chromium-browser` here, btw. Its just a symlink to the normal `chromium` binary which is also set as the mainProgram.

```diff
- then "--set CHROME_BINARY ${lib.getExe' chromium "chromium-browser"}"
+ then "--set CHROME_BINARY ${lib.getExe chromium}"
```

Reply:
Updated this.